hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -450,9 +450,9 @@ public JsonObject toJson() {
         root.add("dirtyEntryCount", dirtyEntryCount);
 
         // keep the contract as milliseconds for latencies sent using Json
-        root.add("totalGetLatencies", NANOSECONDS.toMillis(totalGetLatenciesNanos));
-        root.add("totalPutLatencies", NANOSECONDS.toMillis(totalPutLatenciesNanos));
-        root.add("totalRemoveLatencies", NANOSECONDS.toMillis(totalRemoveLatenciesNanos));
+        root.add("totalGetLatencies", NANOSECONDS.toMillis(totalGetLatencyNanos));
```

Comment:
The names for serializing to json have not been changed due to not breaking anything in MC. But this code will be removed anyway once we move to metrics for sending this data to MC.